web/client/codechecker_client/suppress_file_handler.py

```diff
@@ -0,0 +1,190 @@
+        if src_suppress_format_match:
+            LOG.debug('Match for source code suppress entry format:')
+            src_suppress_format_match = src_suppress_format_match.groupdict()
+            LOG.debug(src_suppress_format_match)
+            suppress_data.append((src_suppress_format_match['bug_hash'],
+                                  src_suppress_format_match['file_name'],
+                                  src_suppress_format_match['comment'],
+                                  src_suppress_format_match['status']))
+            continue
+
+        new_format_match = re.match(new_format, line.strip())
+        if new_format_match:
+            LOG.debug('Match for new suppress entry format:')
+            new_format_match = new_format_match.groupdict()
+            LOG.debug(new_format_match)
+            suppress_data.append((new_format_match['bug_hash'],
+                                  new_format_match['file_name'],
+                                  new_format_match['comment'],
+                                  'false_positive'))
+            continue
+
+        old_format_match = re.match(old_format, line.strip())
+        if old_format_match:
+            LOG.debug('Match for old suppress entry format:')
+            old_format_match = old_format_match.groupdict()
+            LOG.debug(old_format_match)
+            suppress_data.append((old_format_match['bug_hash'],
+                                  u'',  # empty file name
+                                  old_format_match['comment'],
+                                  'false_positive'))
+            continue
+
+        if line.strip() != '':
+            LOG.warning('Malformed suppress line: %s', line)
+
+    return suppress_data
+
+
+def write_to_suppress_file(suppress_file, value, file_name, comment='',
+                           status='false_positive'):
+    comment = comment.decode('UTF-8')
+
+    LOG.debug('Processing suppress file: %s', suppress_file)
+
+    try:
+        with codecs.open(suppress_file, 'r', 'UTF-8') as s_file:
+            suppress_data = get_suppress_data(s_file)
+
+        if not os.stat(suppress_file)[6] == 0:
+            # File is not empty.
+
+            res = filter(lambda x: (x[0] == value and x[1] == file_name) or
+                         (x[0] == value and x[1] == ''),
+                         suppress_data)
+
+            if res:
+                LOG.debug("Already found in\n %s", suppress_file)
+                return True
+
+        s_file = codecs.open(suppress_file, 'a', 'UTF-8')
+
+        s_file.write(value + COMMENT_SEPARATOR +
+                     file_name + COMMENT_SEPARATOR +
+                     comment + COMMENT_SEPARATOR +
+                     status + '\n')
+        s_file.close()
+
+        return True
+
+    except Exception as ex:
+        LOG.error(str(ex))
+        LOG.error("Failed to write: %s", suppress_file)
+        return False
+
+
+def remove_from_suppress_file(suppress_file, value, file_name):
```

Comment:
This function was used only in the [`remove_suppress_bug_id`](https://github.com/Ericsson/codechecker/blob/c0664e3f6448bbd586b3f4d37cf49b9f1c20793c/codechecker_common/suppress_handler.py#L77-L80) function which is removed by this pull request so I think this function can be removed too.

Reply:
I've cleaned up the functions, remove_suppress_bug_id was just removed from the base class, but we did not use it so I've removed it.